modules/govuk_postgresql/manifests/server.pp

```diff
@@ -78,6 +78,7 @@
 
   include govuk_postgresql::mirror
   include postgresql::server::contrib
+  include postgresql::server
```

Comment:
Since you have `class { 'postgresql::server'` above, you don't need this.

Reply:
thank you @rhowe-gds